src/main.rs

```diff
@@ -349,6 +345,14 @@ fn make_discord_message(e: &Event) -> anyhow::Result<Option<serde_json::Value>>
         return Ok(None);
     }
 
+    // Limit description length to make sure it doesn't make scrolling the Discord channel an
```

Comment:
I figure it's okay doing this check at a "global" level? I don't think there should be any perf impact but I would like to measure it going forward once this is more stable.

Reply:
i'd recommend doing this [in EmbedBuilder::description](https://github.com/catppuccin/rockdove/blob/35f265f87e675eea2389c2f3a336820027bff5a6/src/main.rs#L222-L225) instead - it should end up much cleaner!

Reply:
Not sure how this escaped my mind but yup definitely, I'll get around to it tonight if you don't end up implementing it before me!

Reply:
had some other stuff to do (fix the line endings, update dependencies) so i moved the limits into EmbedBuilder and also limited the title & author name while i was in there.